models/validators/BaseValidator.cfc

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright since 2020 by Ortus Solutions, Corp
+ * www.ortussolutions.com
+ * ---
+ * The field under validation must be yes, on, 1, or true. This is useful for validating "Terms of Service" acceptance.
+ */
+component {
```

Comment:
You must add `accessors=true` if not ACF won't grab it.

Reply:
done

Reply:
And I found some accessors typo's in several files